doc/classes/Callable.xml

```diff
@@ -13,21 +13,21 @@
 		    prints(arg1, arg2, arg3)
 
 		func test():
```

Comment:
`test` should be called from `_ready` or even better, should be renamed to `_ready`

Reply:
~~Will do so~~ Actually I think `test` is better since `_ready` might make someone think it needs to be in the `_ready` method, same for C#.